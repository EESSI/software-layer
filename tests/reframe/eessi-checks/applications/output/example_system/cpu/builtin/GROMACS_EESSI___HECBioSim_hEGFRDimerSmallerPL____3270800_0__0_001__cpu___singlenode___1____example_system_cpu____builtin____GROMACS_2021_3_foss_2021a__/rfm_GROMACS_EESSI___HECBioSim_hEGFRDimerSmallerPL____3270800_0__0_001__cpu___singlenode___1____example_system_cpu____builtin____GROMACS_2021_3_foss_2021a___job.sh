#!/bin/bash
#SBATCH --job-name="rfm_GROMACS_EESSI___HECBioSim_hEGFRDimerSmallerPL____3270800_0__0_001__cpu___singlenode___1____example_system_cpu____builtin____GROMACS_2021_3_foss_2021a___job"
#SBATCH --ntasks=128
#SBATCH --ntasks-per-node=128
#SBATCH --cpus-per-task=1
#SBATCH --output=rfm_GROMACS_EESSI___HECBioSim_hEGFRDimerSmallerPL____3270800_0__0_001__cpu___singlenode___1____example_system_cpu____builtin____GROMACS_2021_3_foss_2021a___job.out
#SBATCH --error=rfm_GROMACS_EESSI___HECBioSim_hEGFRDimerSmallerPL____3270800_0__0_001__cpu___singlenode___1____example_system_cpu____builtin____GROMACS_2021_3_foss_2021a___job.err
#SBATCH -p thin
module load GROMACS/2021.3-foss-2021a
curl -LJO https://github.com/victorusu/GROMACS_Benchmark_Suite/raw/1.0.0/HECBioSim/hEGFRDimerSmallerPL/benchmark.tpr
srun gmx_mpi mdrun -nb cpu -s benchmark.tpr
