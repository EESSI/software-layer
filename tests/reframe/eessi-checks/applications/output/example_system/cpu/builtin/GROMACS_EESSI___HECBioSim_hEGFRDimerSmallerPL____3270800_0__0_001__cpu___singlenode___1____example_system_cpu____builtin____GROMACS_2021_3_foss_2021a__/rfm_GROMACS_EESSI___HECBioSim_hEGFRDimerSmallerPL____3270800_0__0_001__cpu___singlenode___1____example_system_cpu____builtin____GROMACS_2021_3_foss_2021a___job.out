
JOB STATISTICS
==============
Job ID: 223863
Cluster: snellius
User/Group: casparl/casparl
State: COMPLETED (exit code 0)
Nodes: 1
Cores per node: 128
CPU Utilized: 02:12:09
CPU Efficiency: 55.310f 03:58:56 core-walltime
Job Wall-clock time: 00:01:52
Memory Utilized: 14.08 GB (estimated maximum)
Memory Efficiency: 6.010f 234.38 GB (1.83 GB/core)
